gdal/doc/source/programs/gdal_calc.rst

```diff
@@ -107,17 +149,23 @@ Add two files together:
 
     gdal_calc.py -A input1.tif -B input2.tif --outfile=result.tif --calc="A+B"
 
+Add a few more files together:
```

Comment:
what does this syntax mean ? That a 3 band 'virtual' dataset is created with input1.tif, input2.tif and input3.tif ? I assume only the first band of those files is taken into account ?

Reply:
No. I've added more examples now to explain it better.

Reply:
> No.

Hum, ok, maybe not a 3 band 'virtual' dataset, but a 3-dim numpy array with the slowest varying dimension being the content of each file ?